src/plots/domain.js

```diff
@@ -69,4 +71,60 @@ module.exports = function(opts, extra) {
         }),
         editType: opts.editType
     };
+
+    if(!opts.noGridCell) {
+        out.row = {
+            valType: 'integer',
+            min: 0,
```

Comment:
> It might be worth adding a `dflt: 0`, so that `domain: {column: 2}` defaults to subplot `xy2`?

Yeah, makes sense - if you have a grid, by default all subplots should go in it. You can always set explicit `domain.x` and `domain.y` anyhow, this is only setting the defaults for those attributes -> https://github.com/plotly/plotly.js/pull/2399/commits/4b43e35587ea990f24f4598113189069878476a1

Reply:
> It might be nice to allow `domain: {row: ''}` to match the 'x'/'x2'/'x3' counter

As discussed offline, I omitted this one (though if you say `''` you'll get `0` now anyway with the change ^^ so the result is the same)